pkgdb2/ui/packages.py

```diff
@@ -878,6 +880,19 @@ def package_unretire(namespace, package, full=True):
                     'error')
                 break
 
+        # Check the review URL
+        bz = APP.config.get('PKGDB2_BUGZILLA_URL')
+        try:
+            int(review_url)
+            review_url = bz + '/' + review_url
+        except (TypeError, ValueError):
+            pass
+        if review_url and bz not in review_url:
```

Comment:
Regarding showing, it's in a different part of the app since this is the review to unretire not the original review we link to in the package view, but I could probably use the same logic here if we want.

For the comment, we could yes. This makes me think we probably need an utility method to check the BZ url in general.